machine-learning/load_test.sh

```diff
@@ -0,0 +1,24 @@
+export MACHINE_LEARNING_CACHE_FOLDER=/tmp/model_cache
+export MACHINE_LEARNING_MIN_FACE_SCORE=0.034 # returns 1 face per request; setting this to 0 blows up the number of faces to the thousands
+export MACHINE_LEARNING_MIN_TAG_SCORE=0.0
+export PID_FILE=/tmp/locust_pid
+export LOG_FILE=/tmp/gunicorn.log
+export HEADLESS=false
+export HOST=127.0.0.1:3003
+export CONCURRENCY=4
+export NUM_ENDPOINTS=3
+export PYTHONPATH=app
+
+gunicorn app.main:app --worker-class uvicorn.workers.UvicornWorker \
```

Comment:
While the app normally just uses `uvicorn`, `gunicorn` has a --daemon flag so it can run in the background.